src/backend/distributed/utils/citus_ruleutils.c

```diff
@@ -209,6 +210,41 @@ AppendOptionListToString(StringInfo stringBuffer, List *optionList)
 
 
 /*
+ * pg_get_sequencedef_string returns the definition of a given sequence. This
+ * definition includes explicit values for all CREATE SEQUENCE options.
+ */
+char *
+pg_get_sequencedef_string(Oid sequenceRelationId)
+{
+	char *qualifiedSequenceName = generate_relation_name(sequenceRelationId, NIL);
```

Comment:
I'll fix. I believe this is violated in several places in this file, FWIW (a quick pass seems to turn up `ReplicateGrantStmt` and `pg_get_indexclusterdef_string` as offenders).